sdk/eventhub/event-hubs/package.json

```diff
@@ -88,7 +88,7 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-amqp": "^1.1.6",
+    "@azure/core-amqp": "^2.0.0-beta.1",
```

Comment:
When using beta versions as dependencies, we shouldnt use the ^ and they must be pinned